idaplugin/rematch/network.py

```diff
@@ -130,8 +141,9 @@ def get_headers(token, json):
   if json:
     headers['Accept'] = 'application/json, text/html, */*'
     headers['Content-Type'] = 'application/json'
-  if token is None and 'token' in config and config['token']:
-    token = config['token']
+  if token is None and token not in config['login']:
```

Comment:
I think this should be `'token'` and not `token`. checking `token in config` has no meaning.